ccdproc/combiner.py

```diff
@@ -546,6 +617,10 @@ def combine(img_list, output_file=None, method='average', weights=None,
         else:
             to_set_in_combiner['scaling'] = scale
 
+    if iraf_minmax_clip:
+        to_call_in_combiner['iraf_minmax_clipping'] = {'nlow': nlow,
+                                                  'nhigh': nhigh}
```

Comment:
one small nitpicky comment here: The ``'nhigh' ...`` could be aligned with the ``'nlow': ...``.